arangod/Aql/Functions.cpp

```diff
@@ -959,7 +959,7 @@ AqlValue mergeParameters(ExpressionContext* expressionContext, transaction::Meth
       builder = arangodb::basics::VelocyPackHelper::merge(builder.slice(), it,
                                                           false, recursive);
     }
-    return AqlValue(builder.slice());
+    return AqlValue(builder.slice(), builder.size());
```

Comment:
why not move the buffer inside ?

Reply:
`Builder` does not provide access to its buffer so that it can be moved.
Am I missing something?

Reply:
you can steal the buffer, or just
```c++
VPackBuffer buffer;
VPackBuilder b(buffer);
```

Reply:
then you can move buffer

Reply:
Sure, but a few lines above the `builder` is exchanged in a call to `VelocyPackHelper::merge(...)`.
So it doesn't make sense to me to assign a fixed buffer before.

Reply:
doesn't `*builder.steal()` work?

Reply:
Yes, this will provide the shared_ptr to the Buffer. Do you think this is worth it?
I will add it then.

Reply:
I don't know, should save a copy in theory no?

Reply:
anyway prob does not matter too much here